CHANGELOG.md

```diff
@@ -28,6 +28,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 - Add `getFieldOption`, `setFieldOption` methods to the FilterInterface
 - Add the `getFilterFieldDescription` method to the AdminInterface
 - Add the `getMaxPageLinks`, `setMaxPageLinks` methods to the PagerInterface
+- Add the `setIsShowMosaicButton`, `hideMosaicButton`, `showMosaicButton` methods to the AbstractAdmin
+- Add the `show_mosaic_button` option to configuration file
```

Comment:
"to configuration~~ file~~" (it might be done in several files, we don't really care)